homeassistant/components/zwave_me/cover.py

```diff
@@ -49,7 +52,7 @@ def open_cover(self, **kwargs):
     def set_cover_position(self, **kwargs: Any) -> None:
         """Update the current value."""
         self.controller.zwave_api.send_command(
-            self.device.id, f"exact?level={str(round(value))}"
+            self.device.id, f"exact?level={str(round(kwargs[ATTR_POSITION]))}"
```

Comment:
The value is already an integer.
```suggestion
            self.device.id, f"exact?level={str(kwargs[ATTR_POSITION])}"
```